lib/consumer_supervisor.ex

```diff
@@ -262,6 +258,23 @@ defmodule ConsumerSupervisor do
     call(supervisor, :count_children)
   end
 
+  @doc """
+  Receives a template to initialize and a set of options.
+
+  This is typically invoked at the end of the init/1 callback of module-based supervisors.
+
+  This function returns a the child specification and the supervisor flags.
+
+  """
+  def init({_, _, _, _, _, _} = template, opts) do
```

Comment:
@maennchen let's keep supporting a list because we can rely on the `Supervisor` definition and signatures. Otherwise stuff like `Supervisor.init` does not work.